projects/knora-ui/src/lib/viewer/values/text-value/text-value-as-string/text-value-as-string.component.html

```diff
@@ -1,22 +1,28 @@
-<span [formGroup]="form">
-    <mat-form-field class="large-field child-value-component" floatLabel="never">
-        <input matInput [formControlName]="'textValue'" class="value" placeholder="Text value" type="text" [readonly]="mode === 'read'">
-    </mat-form-field>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-        <textarea matInput
-                  cdkTextareaAutosize
-                  #autosize="cdkTextareaAutosize"
-                  cdkAutosizeMinRows="1"
-                  cdkAutosizeMaxRows="6"
-                  [formControlName]="'comment'"
-                  class="comment"
-                  placeholder="Comment"
-                  type="text"
-                  [readonly]="mode === 'read'"
-                  spellcheck="false">
-        </textarea>
-    </mat-form-field>
-    <mat-error class="custom-error-message" *ngIf="form.controls.textValue.hasError('valueNotChanged')">
-        Please change the value
-    </mat-error>
-</span>
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <p class="rm-value">{{valueFormControl.value}}</p>
+    <p class="rm-comment" *ngIf="shouldShowComment">{{commentFormControl.value}}</p>
+</div>
+<ng-template #showForm>
+    <span [formGroup]="form">
+        <mat-form-field class="large-field child-value-component" floatLabel="never">
+            <input matInput [formControlName]="'textValue'" class="value" placeholder="Text value" type="text" [readonly]="mode === 'read'">
```

Comment:
see comment above about readonly.

Reply:
Will be done in https://dasch.myjetbrains.com/youtrack/issue/DSP-206